### ISSUE TITLE
Error: connect ECONNREFUSED 127.0.0.1:8090

### ISSUE THREAD

**Issue by tanzeelrana** [input-output-hk/daedalus#649]
[1] [17:41:32:0275] [error] AdaApi::syncProgress error: {
[1]   "stack": "Error: connect ECONNREFUSED 127.0.0.1:8090\n    at Object.exports._errnoException (util.js:1050:11)\n    at exports._exceptionWithHostPort (util.js:1073:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1097:14)",
[1]   "message": "connect ECONNREFUSED 127.0.0.1:8090",
[1]   "code": "ECONNREFUSED",
[1]   "errno": "ECONNREFUSED",
[1]   "syscall": "connect",
[1]   "address": "127.0.0.1",
[1]   "port": 8090
[1] }

**Comment by SebastienGllmt** [comment]
This is a dupe of #608

**Comment by dmitrystas** [comment]
[2018-01-27 15:39:48:0611] [error] CardanoClientApi::syncProgress error: {
  "stack": "Error: connect ECONNREFUSED 127.0.0.1:8090\n    at Object.exports._errnoException (util.js:1026:11)\n    at exports._exceptionWithHostPort (util.js:1049:20)\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1081:14)",
  "message": "connect ECONNREFUSED 127.0.0.1:8090",
  "code": "ECONNREFUSED",
  "errno": "ECONNREFUSED",
  "syscall": "connect",
  "address": "127.0.0.1",
  "port": 8090
}

**Comment by humanoidcreature** [comment]
I've got the same error with the Version 1.1.5813. I've read a lot of issues with this error, so I tried to run the cardano-node.exe manually, to see if that's the problem or not.
It won't run:
`[*production*:INFO:ThreadId 3] [2018-04-28 18:29:55.08 UTC] [Attention] Software is built with wallet part
[*production*:INFO:ThreadId 3] [2018-04-28 18:29:55.08 UTC] using configurations: ConfigurationOptions {cfoFilePath = "lib/configuration.yaml", cfoKey = "default", cfoSystemStart = Nothing, cfoSeed = Nothing}
[node:ERROR:ThreadId 3] [2018-04-28 18:29:55.08 UTC] ConfigurationParseFailure "lib/configuration.yaml" (InvalidYaml (Just (YamlException "Yaml file not found: lib/configuration.yaml")))
cardano-node.exe: ConfigurationParseFailure "lib/configuration.yaml" (InvalidYaml (Just (YamlException "Yaml file not found: lib/configuration.yaml")))`
So I've searched for the configuration.yaml (it was in the root folder) created a lib folder and put it in the config file and run the node again:
`[*production*:INFO:ThreadId 3] [2018-04-28 18:36:26.65 UTC] [Attention] Software is built with wallet part
[*production*:INFO:ThreadId 3] [2018-04-28 18:36:26.65 UTC] using configurations: ConfigurationOptions {cfoFilePath = "lib/configuration.yaml", cfoKey = "default", cfoSystemStart = Nothing, cfoSeed = Nothing}
[node:ERROR:ThreadId 3] [2018-04-28 18:36:26.92 UTC] MissingSystemStartTime
cardano-node.exe: MissingSystemStartTime`
But I don't have any idea what can I do with this error.
